org.flightgear.FlightGear.yaml

```diff
@@ -105,8 +105,8 @@ modules:
   buildsystem: simple
   sources:
   - type: archive
-    url: https://downloads.sourceforge.net/project/flightgear/release-2020.3/FlightGear-2020.3.6-data.tar.bz2
-    sha256: c11a6c6dd596a82dd9bf59326c2ef7fb971ba79a236b4036db633bb6721a41ff
+    url: https://downloads.sourceforge.net/project/flightgear/release-2020.3/FlightGear-2020.3.8-data.txz
+    sha1: 8f919b22422c8ba708bba22c638416b9878031cc
```

Comment:
Please prefer using a sha256 as sha1 is unsafe. You can firs try replacing this line with `sha256: some_foobar_garbage` and the build will give you the correct sha256 saying something like `Expected sha256 some_foobar_garbage but got c11a6c6dd596a82dd9bf59326c2ef7fb971ba79a236b4036db633bb6721a41ff` so here you'll just need to copy paste the sha256 it showed.